go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.0.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/azuremonitorexporter v0.0.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.0.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/honeycombexporter v0.0.0-00010101000000-000000000000
```

Comment:
nit: no practical difference, just to match the others
```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/honeycombexporter v0.0.0
```

Reply:
Perfect - fixed in fa4961a732475b95b6a11705319077519d45dec8. Thanks!